include/avif/avif.h

```diff
@@ -57,7 +57,7 @@ extern "C" {
 // to leverage in-development code without breaking their stable builds.
 #define AVIF_VERSION_MAJOR 1
 #define AVIF_VERSION_MINOR 0
-#define AVIF_VERSION_PATCH 0
+#define AVIF_VERSION_PATCH 1
```

Comment:
Note that I left ` AVIF_VERSION_DEVEL` (in the next line) unchanged (equal to 1) on the main branch.